recipes/liblbfgs/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "liblbfgs" %}
+{% set version = "1.10" %}
+{% set sha = "4158ab7402b573e5c69d5f6b03c973047a91e16ca5737d3347e3af9c906868cf" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/downloads/chokkan/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha }}
+
+build:
```

Comment:
should we add run_exports ?

Reply:
this is new to me. could you detail this a bit?

Reply:
https://conda-forge.org/docs/meta.html#using-conda-build-s-run-exports-feature

The idea is that if you use this lib in your host section, it will be automatically added to the run section. For example here:
https://github.com/conda-forge/zlib-feedstock/blob/master/recipe/meta.yaml#L13

This means we do not need to add zlib explicitly to the run section anymore.

Reply:
done